themes/perun/perun/disco-tpl.php

```diff
@@ -23,165 +24,149 @@
 $this->data['head'] .= '<link rel="stylesheet" media="screen" type="text/css" href="' .
     Module::getModuleUrl('perun/res/css/disco.css') . '" />';
 
-$this->data['head'] .= '<script type="text/javascript" src="' .
-    Module::getModuleUrl('perun/res/js/jquery.livesearch.js') . '"></script>';
-
-$this->data['head'] .= '<script type="text/javascript" src="' .
-    Module::getModuleUrl('discopower/assets/js/suggest.js') . '"></script>';
-
-$this->data['head'] .= Disco::searchScript();
-$this->data['head'] .= Disco::showEntriesScript();
-$this->data['head'] .= Disco::setFocus();
-
-const CONFIG_FILE_NAME = 'module_perun.php';
-
-const ADD_INSTITUTION_URL = 'disco.addInstitution.URL';
-const ADD_INSTITUTION_EMAIL = 'disco.addInstitution.email';
-
-const URN_CESNET_PROXYIDP_IDPENTITYID = 'urn:cesnet:proxyidp:idpentityid:';
-
-const WARNING_TYPE_INFO = 'INFO';
-const WARNING_TYPE_WARNING = 'WARNING';
-const WARNING_TYPE_ERROR = 'ERROR';
-
-$warningIsOn = $this->data['warningIsOn'];
-$warningType = $this->data['warningType'];
-$warningTitle = $this->data['warningTitle'];
-$warningText = $this->data['warningText'];
-
+$warningAttributes = $this->data[Disco::WARNING_ATTRIBUTES];
```

Comment:
It is possible to load 'warningAttributes' from this page? The appropriate configuration file is loaded in this page so we can get this configuration options from it, is't it?

Reply:
It is. In the Disco.php file, this set of data is assigned and thus can be loaded. I wouldn't parse the configuration file once again, as the "warning" configuration uses objects that represent the data set (instead of a plain array).